runtimes/pythonrt/local_runner.go

```diff
@@ -365,16 +368,30 @@ func createVEnv(pyExe string, venvPath string) error {
 	if _, err := io.Copy(file, bytes.NewReader(pyProjectTOML)); err != nil {
 		return fmt.Errorf("copy requirements to %q: %w", file.Name(), err)
 	}
-	file.Close()
 
 	venvPy := path.Join(venvPath, "bin", "python")
-	cmd = exec.Command(venvPy, "-m", "pip", "install", ".[all]")
+	if err := install(venvPy, tmpDir, []string{"-m", "pip", "install", ".[all]"}); err != nil {
+		return fmt.Errorf("install dependencies from %q: %w", file.Name(), err)
+	}
+
+	if err := copyFS(pysdk, tmpDir); err != nil {
+		return err
+	}
+
+	if err = install(venvPy, path.Join(tmpDir, "/py-sdk"), []string{"-m", "pip", "install", "."}); err != nil {
+		return fmt.Errorf("install autokitteh py sdk %w", err)
+	}
+	return nil
+}
+
+func install(pyPath, cwd string, args []string) error {
```

Comment:
Rename to `runInDir` or something like that?

Reply:
I think working directory is standard term in this command line execution stuff